src/huggingface_hub/hf_api.py

```diff
@@ -7655,6 +7684,368 @@ def delete_collection_item(
+    ) -> List[AccessRequest]:
+        """
+        Get accepted access requests for a given gated repo.
+
+        An accepted request means the user has requested access to the repo and the request has been accepted. The user
+        can download any file of the repo and access the community tab. If the approval mode is automatic, this list
+        should contains by default all requests. Accepted requests can be cancelled or rejected at any time using
+        [`cancel_access_request`] and [`reject_access_request`]. A cancelled request will go back to the pending list
+        while a rejected request will go to the rejected list. In both cases, the user will lose access to the repo.
+
+        For more info about gated repos, see https://huggingface.co/docs/hub/models-gated.
+
+        Args:
+            repo_id (`str`):
+                The id of the repo to get access requests for.
+            repo_type (`str`, *optional*):
+                The type of the repo to get access requests for. Must be one of `model`, `dataset` or `space`.
+                Defaults to `model`.
+            token (`str`, *optional*):
+                A valid authentication token (see https://huggingface.co/settings/token).
+
+        Returns:
+            `List[AccessRequest]`: a list of [`AccessRequest`] objects. Each time contains a `username`, `email`,
+            `status` and `timestamp` attribute. If the gated repo has a custom form, the `fields` attribute will
+            be populated with user's answers.
+
+        Raises:
+            `HTTPError`: HTTP 400 if the repo is not gated.
+
+        Example:
+        ```py
+        >>> from huggingface_hub import list_accepted_access_requests
+
+        >>> requests = list_accepted_access_requests("meta-llama/Llama-2-7b")
+        >>> len(requests)
+        411
+        >>> requests[0]
+        [
+            AccessRequest(
+                username='clem',
+                fullname='Clem 🤗',
+                email='***',
+                timestamp=datetime.datetime(2023, 11, 23, 18, 4, 53, 828000, tzinfo=datetime.timezone.utc),
+                status='accepted',
+                fields=None,
+            ),
+            ...
+        ]
+        ```
+        """
+        return self._list_access_requests(repo_id, "accepted", repo_type=repo_type, token=token)
+
+    @validate_hf_hub_args
+    def list_rejected_access_requests(
+        self, repo_id: str, *, repo_type: Optional[str] = None, token: Optional[str] = None
+    ) -> List[AccessRequest]:
+        """
+        Get rejected access requests for a given gated repo.
+
+        A rejected request means the user has requested access to the repo and the request has been explicitly rejected
+        by a repo owner (either you or another user from your organization). The user cannot download any file of the
+        repo and cannot access the community tab. Rejected requests can be accepted or cancelled at any time using
```

Comment:
> and cannot access the community tab

Are you sure about this? I thought all users had access to community tab

Reply:
Wow sorry, don't know from where I invented that. I think I'm hallucinating facts :face_with_spiral_eyes:

Reply:
Changed in https://github.com/huggingface/huggingface_hub/pull/1905/commits/31a35dab75e7bfb4b75c26a0d904df887b007780

Reply:
you used a LLM copilot maybe

Reply:
LM stands for Lucain Model